tailwind.config.ts

```diff
@@ -1,8 +1,12 @@
 import type { Config } from "tailwindcss";
 
 const config: Config = {
-  content: ["./src/app/**/*.{js,ts,jsx,tsx,mdx}"],
-  theme: {},
-  plugins: [],
+    content: ["./src/app/**/*.{js,ts,jsx,tsx,mdx}"],
+    theme: {
+        screens: {
+            sm: "350px",
+        },
+    },
```

Comment:
Please do not override the existing tailwind breakpoints.

Instead opt to add a new one. E.g. `xs`

```suggestion
    theme: {
        extend: {
            screens: {
                xs: "350px",
            },
        },
    },
```